packages/chat-core-zendesk/src/infra/ChatCoreZendeskImpl.ts

```diff
@@ -63,7 +63,12 @@ export class ChatCoreZendeskImpl {
    * mode on the first invocation. Subsequent calls to this method will create a
    * new conversation session.
    */
-  async init(messageRsp: MessageResponse): Promise<void> {
+  async init(messageRsp: MessageResponse): Promise<string> {
```

Comment:
this update causes mismatch signature with ChatCoreZendesk interface. We can either update the interface or leave it as is and rely on `getSession()` function to grab the relevant details

Reply:
Updated the interface!